src/Locator/SkipAwareFilesystemFeatureLocator.php

```diff
@@ -0,0 +1,69 @@
+<?php
+
+/*
+ * This file is part of the SkipExtension package.
+ *
+ * (c) Kamil Kokot <kamil@kokot.me>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace FriendsOfBehat\SkipExtension\Locator;
+
+use Behat\Testwork\Specification\Locator\SpecificationLocator;
+use Behat\Testwork\Suite\Suite;
+use FriendsOfBehat\SkipExtension\Iterator\SkipAwareFeatureIterator;
+
+/**
+ * @author Mateusz Zalewski <mateusz.p.zalewski@gmail.com>
```

Comment:
@FriendsOfBehat extensions usually does not contain author blocks, maintaining them is quite hard and whole history about authors is still in Git repository